trex-scripts/tests/priority_scheduling.py

```diff
@@ -0,0 +1,113 @@
+# SPDX-FileCopyrightText: Copyright 2020-present Open Networking Foundation.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import os
+import json
+from argparse import ArgumentParser
+from datetime import datetime
+from scapy.layers.all import IP, TCP, UDP, Ether
+
+from lib.base_test import StatelessTest
+from lib.utils import list_port_status,get_readable_port_stats
+from lib.xnt import analysis_report_pcap
+from trex_stl_lib.api import STLPktBuilder, STLStream, STLTXCont, STLTXSingleBurst, STLFlowLatencyStats
+
+
+SOURCE_MAC = "00:00:00:00:00:01"
+DEST_MAC   = "00:00:00:00:00:02"
+
+SENDER_PORT   = [0]
+RECEIVER_PORT = [1]
+
+
+class PriorityScheduling(StatelessTest):
```

Comment:
Ideally, this test should automate testing with and without priorities, but I guess before we can do that we need to wait for @Yi-Tseng to complete the integration with the p4runtime library that is happening in fabric-tna. So maybe we can do that in a later PR.

Reply:
Nice. Got it.